src/com/biocollapse/view/ConfigHistoryPanel.java

```diff
@@ -0,0 +1,117 @@
+// Authors: Johann
+package src.com.biocollapse.view;
+
+import java.awt.BorderLayout;
+import java.awt.Dimension;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.ObjectInputStream;
+import java.util.HashMap;
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JList;
+import javax.swing.JPanel;
+import javax.swing.SwingConstants;
+import javax.swing.border.TitledBorder;
+import src.com.biocollapse.model.Config;
+import src.com.biocollapse.util.GlobalConfig;
+
+public class ConfigHistoryPanel extends JPanel {
+
+    private final ConfigPanel configPanel;
+    private HashMap<String, File> files = new HashMap<>();
```

Comment:
I had something similar in the Map but the prof. didnt like it. 

He said something about not needing the file references in memory or somethin...

Reply:
haha. always these C profs ranting about memory.

I can make it a bit different. But this is a super valid use case. Another thing: which maybe some C People dont want to agree with: These days code doesnt have to be the most memory efficient anymore. Because the computers are fast and good enough for it not to make a difference.

But heyy i will change it 😂

Reply:
Hmm actually it needs to be this way. Maybe i will just remove my name as author 🫣

Reply:
Ahh yes the other one was static.
That probably changes some stuff since your memory stuff is "thrown away" afterwards...

Reply:
Okay yeaa. Thanks for the feedback nonetheless @ColorfulBisuketto ! Can u approve it in that case?